### PR TITLE
add functionality to tick and cross buttons on location selector

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #35

### CODE REVIEW

File: app/public/map.js [hunk 1]
```diff
@@ -93,8 +93,20 @@ var nazarethNuggets = (function () { // eslint-disable-line
...
     mymap.locate({setView: true})
   })
+
+  var tickButton = document.querySelector('.location-select-tick')
+  tickButton.addEventListener('click', function (e) {
+    locationSelector.classList.toggle('visible')
+    // toggle form tab not visible -- future goooal
+  })
+
+  var infoCrossBtn = document.querySelector('.location-select-cross')
```
Comment (Karyum) [inline app/public/map.js]: maybe we should change the variable name to something else cause i thought  infoCrossBtn is the cross button for the info tab

Comment (Suuuuha) [inline app/public/map.js]: can u write in english ?

Comment (Suuuuha) [reply app/public/map.js]: i guess u're right , changed it :+1:

**Review by Karyum — APPROVED** [review]

File: app/public/map.js [hunk 2]
```diff
@@ -93,8 +93,20 @@ var nazarethNuggets = (function () { // eslint-disable-line
 
   addIconsToMap(nuggets)
 
+  var locationSelector = document.querySelector('.location-select-display')
```
Comment (mattlub) [inline app/public/map.js]: why not call this locationSelectDisplay?

File: app/public/index.html [hunk 3]
```diff
@@ -47,7 +47,7 @@ <h3 class="nugget-title">
     <!-- ADD-FORM -->
 
     <!-- LOCATION-SELECT -->
-    <div class="location-select-display">
+    <div class="location-select-display visible">
```
Comment (mattlub) [inline app/public/index.html]: shouldn't start visible right?

**Review by mattlub — CHANGES_REQUESTED** [review]
sorry for another change request 😁

File: app/public/index.html (hunk 3 above)
Comment (Suuuuha) [reply app/public/index.html]: yeah it was there for testing and forgot to remove .. thnx

**Review by mattlub — APPROVED** [review]

**Review by HilbertSpitzer — APPROVED** [review]

**Pull request merged**